Automattic/jetpack
# Admin - General tab: remove 3 cards at the bottom / Show site plan info.

closes #3832
closes #3833  

## Removes cards in General tab.
```
Summary Report Settings
Import Jetpack Feature Configuration
Widget Settings
```
To test, go to Jetpack > Dashboard > General tab and make sure these cards aren't there anymore.

## Fills Jetpack Add-ons card with plan info.
To test:
1. apply D1857
2. in your wporg sandbox, edit `jetpack.php` and set `JETPACK__WPCOM_JSON_API_HOST` to your wpcom sandbox (must be permanently pointed).
3. go to Jetpack > Dashboard > General tab and click on the Jetpack Add-ons arrow, after fetching the data, it should display "Free" or whatever your plan is.

Site in dev mode. Can be seen in a local server for example.
<img width="423" alt="dev mode" src="https://cloud.githubusercontent.com/assets/1041600/15446309/d20eb584-1ef2-11e6-9abd-3224bf71afff.png">

While data is loading...
<img width="464" alt="after site data is loaded" src="https://cloud.githubusercontent.com/assets/1041600/15443925/63bb216a-1ec2-11e6-8fb8-e6617c1abda2.png">

If data can't be loaded. Can be seen if D1857 isn't applied.
<img width="404" alt="data error" src="https://cloud.githubusercontent.com/assets/1041600/15446310/f57ad16a-1ef2-11e6-859e-a0029851bb0e.png">

After data is correctly loaded and displayed.
<img width="409" alt="while fetching site data" src="https://cloud.githubusercontent.com/assets/1041600/15443924/63b41ac8-1ec2-11e6-8b8c-cae3ea4c463e.png">
